src/addons/Responsive/Responsive.js

```diff
@@ -46,6 +53,10 @@ export default class Responsive extends Component {
     onUpdate: PropTypes.func,
   }
 
+  static defaultProps = {
+    getWidth: () => (isBrowser() ? window.innerWidth : 0),
```

Comment:
I would think if there's no window at all, `null` might be more appropriate.  Thoughts?